utilities/constants.py

```diff
@@ -615,6 +615,7 @@ class StorageClassNames:
     TRIDENT_CSI_FSX = "trident-csi-fsx"
     IO2_CSI = "io2-csi"
     IBM_SPECTRUM_SCALE = "ibm-spectrum-scale-sample-uid-gid-107"
+    GPFS = "ibm-spectrum-scale-sample"
```

Comment:
Hi @qwang1, you added the `IBM_SPECTRUM_SCALE` some time ago, so a couple of questions:
- are you using it?
- is the name still relevant? Can we now name it "ibm-spectrum-scale-sample"?